libcloud/compute/drivers/softlayer.py

```diff
@@ -278,6 +278,8 @@ def create_node(self, **kwargs):
         :type       ex_os: ``str``
         :keyword    ex_keyname: The name of the key pair
         :type       ex_keyname: ``str``
+        :keyword    post_uri: Post-install script to run after reload
```

Comment:
Some more documentation for this argument (and ideally example in the docs) would also be nice.

Right now it's not really clear what the value should contain - it seems like it should be a full URL to a file or something else?